spp_change_request/models/mixins/source_mixin.py

```diff
@@ -603,7 +602,7 @@ def _copy_from_group_member_ids(self, group_ref_field, group_id_field, skip_head
                         "kind_ids": kind_ids,
                         "new_relation_to_head": None,
                     }
-                    self.env["pds.change.request.src.grp"].create(group_members)
+                    self.env["spp.change.request.src.grp"].create(group_members)
```

Comment:
@gonzalesedwin1123 upon checking, there is no existing model with the name `spp.change.request.src.grp ` in OpenSPP

Reply:
Yes but I plan to keep this function as I need to scan all that might still be using this. I will have this function removed once I am done checking everything that might be referring to this function.

Reply:
ahh, i see, I think it is better to comment it as well, then add a TODO note, something like below code

```
# TODO: Investigate this
# self.env["spp.change.request.src.grp"].create(service_points)
```